ROSCO/src/ReadSetParameters.f90

```diff
@@ -390,6 +390,14 @@ SUBROUTINE ReadControlParameterFileSub(CntrPar, zmqVar, accINFILE, accINFILE_siz
         CALL ParseInput(UnControllerParameters,CurLine,'PA_CornerFreq',accINFILE(1),CntrPar%PA_CornerFreq,ErrVar)
         CALL ParseInput(UnControllerParameters,CurLine,'PA_Damping',accINFILE(1),CntrPar%PA_Damping,ErrVar)
         CALL ReadEmptyLine(UnControllerParameters,CurLine)   
+
+        !------------ AWC input ------------
+        CALL ReadEmptyLine(UnControllerParameters,CurLine)  
+        CALL ParseInput(UnControllerParameters,CurLine,'AWC_NumModes',  accINFILE(1),CntrPar%AWC_NumModes, ErrVar)
+        CALL ParseAry(  UnControllerParameters,CurLine,'AWC_n',         CntrPar%AWC_n,     CntrPar%AWC_NumModes, accINFILE(1), ErrVar)
+        CALL ParseAry(  UnControllerParameters,CurLine,'AWC_omega',     CntrPar%AWC_omega, CntrPar%AWC_NumModes, accINFILE(1), ErrVar)
+        CALL ParseAry(  UnControllerParameters,CurLine,'AWC_amp',       CntrPar%AWC_amp,   CntrPar%AWC_NumModes, accINFILE(1), ErrVar)
+        CALL ParseAry(  UnControllerParameters,CurLine,'AWC_clockangle',CntrPar%AWC_clockangle,CntrPar%AWC_NumModes, accINFILE(1), ErrVar)
```

Comment:
Do you have a DISCON file that corresponds to these added inputs?

Reply:
This was added in a more recent commit